squad_client/commands/submit_tuxbuild.py

```diff
@@ -104,7 +105,7 @@ def _build_metadata(self, build):
         metadata.update({"make_kernelversion": metadata.get("kernel_version")})
 
         # add config file to the metadata
-        metadata["config"] = f"{metadata.get('download_url')}/config"
+        metadata["config"] = f"{metadata.get('download_url')}config"
```

Comment:
What happens if tuxsuite guys remove the '/' from the download_url ?
Thats why the extra '/' was added.

Reply:
Fixed it with urljoin